cve_bin_tool/parsers/__init__.py

```diff
@@ -5,6 +5,7 @@
     "Parser",
     "java",
     "javascript",
+     "rust",
```

Comment:
```suggestion
    "rust",
```

Looks like an off-by-one indent error.